src/util.h

```diff
@@ -82,28 +82,34 @@
 
 #else
 
+void* (*git__malloc_func)(size_t size) = malloc;
+void* (*git__calloc_func)(size_t nitems, size_t size) = calloc;
```

Comment:
I think we'll want to have `typedef`s for these, so we don't need to repeat them here and over in `src/settings.c`.